test/js/node/http/node-http.test.ts

```diff
@@ -744,6 +750,119 @@ describe("node:http", () => {
         req.end();
       });
     });
+    it(`should allow piping multiple chunks to collector stream`, done => {
+      const chunkSize = 1024 * 16; // 16 kb
+      const testData = Buffer.from(Array(chunkSize * 5).fill("a"));
+
+      const createStream = () =>
+        new Response(
+          new ReadableStream({
+            async start(controller) {
+              const chunksNum = Math.ceil(testData.length / chunkSize);
+
+              for (let i = 0; i < chunksNum; i++) {
+                const start = i * chunkSize;
+                controller.enqueue(testData.subarray(start, start + chunkSize));
+              }
+
+              controller.close();
+            },
+          }),
+          {
+            headers: {
+              "content-length": testData.length.toString(),
```

Comment:
This content-length header will be completely ignored by the server

Reply:
Hmmm i dont know why that would be an issue? From what I read the code content-length is only used to decide whether body should be parsed or not.

Reading body should be the same for requests and responses.